contrib/demo/workspaceKubectlPlugin-script/workspaceKubectlPlugin-removeClusters.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Instead of checking in empty files, just don't call this unless it exists?

Reply:
Will do. Required change has been done in PR #381